src/Neo/VM/Helper.cs

```diff
@@ -218,7 +264,7 @@ public static ScriptBuilder EmitPush(this ScriptBuilder builder, object obj)
                     builder.EmitPush(data);
                     break;
                 case char data:
-                    builder.EmitPush((ushort)data);
+                    builder.EmitPush(data);
```

Comment:
Can we add a test for it?

Reply:
https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/builtin-types/char